.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -126,7 +126,13 @@ const uploadPipeline = (pipelineContent: string | object) => {
       pipeline.push(getPipeline('.buildkite/pipelines/pull_request/deploy_cloud.yml'));
     }
 
-    if (GITHUB_PR_LABELS.includes('ci:build-webpack-bundle-analyzer')) {
+    if (
+      (await doAnyChangesMatch([/.*stor(ies|y).*/])) ||
+      GITHUB_PR_LABELS.includes('ci:build-storybooks')
+    ) {
+      pipeline.push(getPipeline('.buildkite/pipelines/pull_request/storybooks.yml'));
```

Comment:
Looks like you're missing a `}`

Reply:
Thanks - fixed with c8518f7162a6096b0b0f947fa9259a5c4bc25bce